gateway/1.5/modules/ROOT/pages/policies-pdk-architecture.adoc

```diff
@@ -2,4 +2,41 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+The Policy Development Kit (PDK) consists of the:
```

Comment:
I suggest adding more information about the purpose of this PDK. For example:
The Policy Development Kit (PDK) is a set of templates that enable who to do what, then follow with the content from line 7 and on. 
Overall, I think this is a good overview